content/applications/hr/payroll.rst

```diff
@@ -332,23 +345,24 @@ Personal Info`.
    :align: center
    :alt: Personal information that appear on employee cards to enter.
 
-To edit an entry, select it from the list. Then, click the :guilabel:`Edit` button, and modify the
-entry. When done, click :guilabel:`Save` or :guilabel:`Discard` to save the information or cancel
-the edits.
+To edit an entry, select it from the list, and modify the entry. To make a new entry, click the
+:guilabel:`New` button.
 
-.. image:: payroll/personal-new.png
-   :align: center
-   :alt: New personal information entry.
+The required fields, aside from entering the :guilabel:`Information` name, are
+:guilabel:`Related Field` and :guilabel:`Category`. Select a :guilabel:`Related Field` from the
+drop-down menu that best describes what kind of information this entry is, and where it is going to
+be stored in the backed. Select a :guilabel:`Category` from the drop-down menu that the information
+should be under, such as :guilabel:`Address` or :guilabel:`Personal Documents`.
 
 The two most important fields on the personal info form are :guilabel:`Is Required` and
 :guilabel:`Display Type`. Checking the :guilabel:`Is Required` box makes the field mandatory on the
-employee's card.
+employee's card. The :guilabel:`Display Type` drop-down menu allows for the information to be
+entered in a variety of ways, from a :guilabel:`Text` box, to a customizable :guilabel:`Radio`
+button, a :guilabel:`Checkbox`, a :guilabel:`Document`, and more.
```

Comment:
This paragraph isn't in line with the rest of the sections, the others are listed. I like the detail in this section though!

Reply:
Zac usually prefers explanations like this, so may be good to go off of and model the other sections like this. They lacked depth.